dxr/plugins/xpidl/tests/test_menus_and_filters/test_filters.py

```diff
@@ -0,0 +1,28 @@
+from dxr.testing import DxrInstanceTestCase
+
+
+class IdlFilterQueryTest(DxrInstanceTestCase):
+    """Test query filters for Idl structure."""
+
+    def test_derived(self):
+        self.found_line_eq('+derived:nsISupports',
```

Comment:
No particular reason; this was just an example of doing it that way on the first case and then copying it to the others. I'll remove them, since they're not qualified.